.yamato/upm-ci.yml

```diff
@@ -135,7 +135,15 @@ test_trigger:
 {% if use_autodesk_fbx_submodule_for_testing %}
     - .yamato/package-test-utr.yml#test_{{ platform.name }}_{{ editor.version }}_using_autodesk_fbx_submodule
 {% else %}
-    - .yamato/upm-ci.yml#test_{{platform.name}}_{{editor.version}}
+    - .yamato/upm-ci.yml#test_{{ platform.name }}_{{ editor.version }}
+{% endif %}
+{% endfor %}
+{% endfor %}
+# Only run clean console tests on Win for PR trigger
+{% for platform in platforms %}
+{% for editor in clean_console_test_editors %}
+{% if platform.name == "win" %}
```

Comment:
would it be possible to remove this if statement and the platform for loop? e.g.
```
{% for editor in clean_console_test_editors %}
    - .yamato/clean-console-test.yml#clean_console_test_win_{{ editor.version }}
{% endfor %}
```

Reply:
Good idea, I will do it